.github/workflows/build.yml

```diff
@@ -9,62 +9,37 @@ on:
   workflow_dispatch:
 
 jobs:
-  unix-build:
+  build:
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
-        os: [ubuntu-18.04, ubuntu-latest]
-        emacs: [26.1, 26.2, 26.3, 27.1, 27.2, snapshot]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        emacs-version: [26.1, 26.2, 26.3, 27.1, 27.2, 28.1, snapshot]
 
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
-      - uses: actions/setup-python@v2
-        with:
-          python-version: "3.6"
-          architecture: "x64"
-
       - uses: purcell/setup-emacs@master
+        if: matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest'
         with:
-          version: ${{ matrix.emacs }}
-
-      - uses: conao3/setup-cask@master
-        with:
-          version: 0.8.4
-
-      - name: paths
-        run: |
-          echo "$HOME/.cask/bin" >> $GITHUB_PATH
-
-      - name: Run a multi-line script
-        run: |
-          emacs --version
-          make test
+          version: ${{ matrix.emacs-version }}
 
-  windows-build:
-    runs-on: windows-latest
-    strategy:
-      matrix:
-        emacs: [26.1, 26.2, 26.3, 27.1, 27.2, snapshot]
-
-    steps:
-      - uses: actions/checkout@v2
-
-      - uses: actions/setup-python@v2
+      - uses: jcs090218/setup-emacs-windows@master
+        if: matrix.os == 'windows-latest'
         with:
-          python-version: "3.6"
-          architecture: "x64"
+          version: ${{ matrix.emacs-version }}
 
-      - uses: jcs090218/setup-emacs-windows@master
+      - uses: actions/setup-node@v2
         with:
-          version: ${{ matrix.emacs }}
+          node-version: '14'
```

Comment:
Let's bump this to 16. Then we have revisit that a little further in the future

Reply:
Bumped!